_profiles/Sida-Lillian.md

```diff
@@ -8,13 +8,13 @@ hub: CC4D
 languages: "English (Fluent spoken and written), Bari/Kakwa (Fluent spoken and spoken), Arabic (good spoken only), Lugbara (fair spoken only)"
 mail: sidalilian@proton.me
 phone: "+256 (0) 782739162"
-whatsapp: "+256782739162"
+whatsapp: 
 website: 
 telegram: "+256782739162"
-github: Sida1990
+github: github.com/sidalillian
 linkedin: 
 twitter: 
-facebook: Lily Savio
+facebook: [Lily Savio](https://www.facebook.com/profile.php?id=100074066215635)
```

Comment:
this I have to check locally

Reply:
Or @MartinSchott can you tell me if the URL needs to be separated here for the button to work?